hack/go-build-bundle.sh

```diff
@@ -24,13 +24,13 @@ BUNDLE_NAME=rhpam-7/${BUNDLE}
 CSV=businessautomation-operator.${VERSION}.clusterserviceversion.yaml
 if [[ ${DEV} == true ]]; then
     OLMDIR=deploy/olm-catalog/dev
-    BUNDLE_NAME=quay.io/tchughesiv/${BUNDLE}
+    BUNDLE_NAME=quay.io/${USERNAME}/${BUNDLE}
 fi
 VERDIR=${OLMDIR}/${VERSION}
 CRD=kieapp.crd.yaml
-if (( $(echo "${VERSION} 7.9.0" | awk '{print ($1 < $2)}') )); then
-    CRD=kieapp.crd.v1beta1.yaml
-fi
+#if (( $(echo "${VERSION} 7.9.0" | awk '{print ($1 < $2)}') )); then
+#    CRD=kieapp.crd.v1beta1.yaml
+#fi
```

Comment:
Why are we keeping this commented code?

Reply:
I commented it out to be able to build the bundle image, it was failing, will revert.